kitchen.yml

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
@mcgege any reason why is it without dot? (Usually it's `.`kitchen.yml)

Reply:
That seems to have changed - without dot is now prefered:
https://kitchen.ci/docs/getting-started/kitchen-yml/

Reply:
Wow, thanks!